subsys/bluetooth/controller/ll_sw/lll_scan_aux.h

```diff
@@ -4,6 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#define AUX_OFFSET_IS_VALID(offset_us, win_size_us, pdu_us) \
+		(((offset_us) + (win_size_us)) >= ((pdu_us) + (EVENT_MAFS_US)))
```

Comment:
```suggestion
#define AUX_OFFSET_IS_VALID(_offset_us, _win_size_us, _pdu_us) \
		(((_offset_us) + (_win_size_us)) >= ((_pdu_us) + (EVENT_MAFS_US)))
```

Suggest to prefix the macro parameters with `_` as the names of the variables are quite common in the controller, and to ensure that we don't accidentally run into any issues with the naming.